core/pkg/service/sync/server.go

```diff
@@ -18,11 +18,13 @@ import (
 )
 
 type Server struct {
-	server        *http.Server
-	metricsServer *http.Server
-	Logger        *logger.Logger
-	handler       *handler
-	config        iservice.Configuration
+	server           *http.Server
+	metricsServer    *http.Server
+	Logger           *logger.Logger
+	handler          *handler
+	config           iservice.Configuration
+	grpcServer       *grpc.Server
+	readinessEnabled bool
```

Comment:
I suggest changing the name since `readinessEnabled` convey a different purpose than what is used for in the code 
```suggestion
	serverReady bool
```

Reply:
I do not find `serverReady` as a good name here.. I would like to stick to the readiness enabled

Reply:
you're right, this is only for the metric part. Maybe `metricServerReady` is the most descriptive name since also not all traffic might be allowed later on